bin/run.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+const chalk = require("chalk");
+const program = require("commander");
+const Versions = require("../lib/versions");
+
+console.log(``);
+console.log(`  ${chalk.black(chalk.bgYellow("  ~~~~~~~~~~  "))}`);
+console.log(`  ${chalk.black(chalk.bgYellow("   🍫  kaba    "))}`);
+console.log(`  ${chalk.black(chalk.bgYellow("  ~~~~~~~~~~  "))}`);
+console.log(``);
+
+program
+    .option('--dev, -d', 'enables debug, file watchers, linting and source maps')
+    .option('--debug', 'enables debug builds (non-minified and with env `development`)')
```

Comment:
Generally speaking: Do we really have a scenario where we'd need the `--dev` flag independent of the `--debug` flag?

I'd argue that most of the time if you want either one, you want both of them. I think of a case where I'd wanted a mix of both.

Reply:
Yes, because you might want to build a debug build without starting a watcher or thrashing your CLI with linting messages.